Authentication/ForgotPassword.html

```diff
@@ -0,0 +1,30 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Blithe quiz app</title>
+    <link rel="stylesheet" href="./Authentication.css">
+</head>
+<body>
+    <section class="login">
+        <div class="col-7 login-img-container">
+            <img src="../Assets/Collage.jpg"
+                alt="login-img" />
+        </div>
+        <div class="col-7 login-content-container">
+            <div class="login-form">
+                <img src="../Assets/Blithe-logo.png" class="blithe-logo" alt="blithe-logo"/>
+                <form>
+                    <p class="heading-md heading">Forgot Password</p>
+                    <div class="float-container">
+                        <label for="floatField" class="input-lbl">Email</label>
+                        <input type="email" name="">
```

Comment:
you can use 'required' attribute for input validation before form submit

Reply:
ok